arbitrum-docs/node-running/running-a-validator

```diff
@@ -0,0 +1,301 @@
+# Running a Validator
+
+Instructions for configuring a deployment for a Nitro node to be run as a validator, on either the Arbitrum One or Arbitrum Nova chain. 
```

Comment:
Maybe let's add a quick definition for "validator" here; something like, "A validator is an Arbitrum full node that also tracks the status of the chain's assertions on L1."